sdk/search/Azure.Search.Documents/src/Models/Suggester.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core;
+
+namespace Azure.Search.Documents.Models
+{
+    [CodeGenSchema("Suggester")]
+    public partial class Suggester
+    {
+        private const string AnalyzingInfixMatching = "analyzingInfixMatching";
```

Comment:
Nit - we have a constants file for things like this.  If we're planning to remove it after a bug gets fixed, then maybe it's easier to leave here in the short term.

Reply:
This is the only place this gets used and is private. Why locate unrelated, non-shared things to a separate file? Guidance across many code bases is to put shared constants in a common file, sure, but not isolated usage. That's what we also did in KV. How bad do you want this?

Reply:
Besides, even if a constant was repeated (if that's what you're trying to avoid), the string literal will still be stamped into the PE/COFF once - not N times (per usage).

Reply:
I don't feel strongly, but having everything in a constants file can make it easier to quickly scan for weird cases like this.